lib/rucio/tests/test_did_meta_plugins.py

```diff
@@ -222,6 +224,97 @@ def test_list_did_meta(self):
         assert [tmp_dsn4] == results
 
 
+class TestDidMetaMongo(unittest.TestCase):
+
+    def setUp(self):
+        self.session = get_session()
+        if config_get_bool('common', 'multi_vo', raise_exception=False, default=False):
+            self.vo = {'vo': get_vo()}
+        else:
+            self.vo = {}
+        self.tmp_scope = InternalScope('mock', **self.vo)
+        self.root = InternalAccount('root', **self.vo)
+
+        self.mongo_meta = MongoDidMeta(
+            host = 'mongo',
```

Comment:
@maany could I get your direction on this please?

For testing locally i've added a mongo container service to the docker-compose files and added an `extra_host` (mongo) for this service inside the dev container.

What is the best way to handle this in terms of the CI pipeline?